### DISCUSSION TITLE
How config loadBalancerPolicy with Gateway ?

### DISCUSSION THREAD

**Discussion by lou-lan** [kubernetes-sigs/gateway-api#912] [Q&A]
e.g `RoundRobin`

**Comment by jpeach** [comment]
There's no direct way to do this in the current API. If you experiment with [policy attachment](https://gateway-api.sigs.k8s.io/v1alpha2/references/policy-attachment/), can you update this thread with feedback?

**Reply by lou-lan** [reply]
I can't read this document very well. Does it mean that each ingress controller developer can implement a different policy definition?

**Comment by hbagdi** [comment]
A follow-up question, @lou-lan, are you trying to implement Gateway API or are you using a software where you have encountered this issue?

**Reply by lou-lan** [reply]
I am a user of this software.

**Reply by jpeach** [reply]
@lou-lan Which implementation are you using? Maybe we can summon someone from that community to give some advice.

**Reply by lou-lan** [reply]
> @lou-lan Which implementation are you using? Maybe we can summon someone from that community to give some advice.

[Contour](https://github.com/projectcontour/contour).

**Reply by youngnick** [reply]
Thanks @lou-lan, I'm a maintainer on Contour, and there's no current way to do this with the Gateway API. We do have Contour support for this, but it requires our CRD, HTTPProxy, to do for now. We are aiming to bring Gateway API support to parity with HTTPProxy one way or another, but there's a long road to get there.

Please feel free to ping me on Kubernetes Slack, or to log a Contour issue and we can talk more details there (I'm @youngnick there as well).

**Reply by lou-lan** [reply]
Thanks.
